FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/Models/FhirToDataLakeAzureStorageJobInfo.cs

```diff
@@ -71,5 +72,13 @@ private string TryParseAsFhirToDataLakeProcessingJobInputData()
 
             return null;
         }
+
+        private static string RemoveJobVersionFromInput(object inputData)
```

Comment:
Maybe we can rename to GetIdentifierText() and comment why we need to exclude version here.

Reply:
Sure, resolved.